screens/ProductionScreens/BudgetAddExpense.jsx

```diff
@@ -241,10 +285,42 @@ export default function BudgetAddExpenseScreen({ navigation, route }) {
         );
     };
 
+    const handleDelete = async () => {
+        try {
+            await runTransaction(
+                dbRef(
+                    db,
+                    `budgets/${budget}/${
+                        isPlaceholder ? "placeholderExpenses" : "nonPlaceholderExpenses"
+                    }`
+                ),
+                (currentValue) => {
+                    if (currentValue) {
+                        return Number(currentValue) - Number(originalCost);
+                    }
+                    return -1 * Number(originalCost);
+                }
+            );
+            const receiptRef = ref(storage, `receipts/${id}${originalReceiptExtension}`);
+            await deleteObject(receiptRef);
+            await set(dbRef(db, `expenses/${id}`), null);
```

Comment:
does this actually delete the whole entry or just set the value to null?

Reply:
I believe setting it to null is equivalent, it just removes that branch of the tree :)